web/src/app/add-stuff/add-stuff.component.html

```diff
@@ -2,7 +2,7 @@ <h1>Add my stuff</h1>
 <form [formGroup]="addStuff" (ngSubmit)="addStuff.valid && save()">
   <div class="form-group">
     <label for="name">Name</label>
-    <input class="form-control" formControlName="name" id="name" required trim="blur"/>
```

Comment:
Name is required attribute for a stuff. Why do you think it's not?

Reply:
This attribute is no longer needed since we've added `Validators.required` into this reactive form. The tests are successful and the form itself works right.

Reply:
Make a sense, my bad.  :)